packages/core/src/Cardano/types/CardanoNode.ts

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { CardanoNodeError, CardanoNodeNotInitializedError } from '../types';
+import { ConnectionConfig } from '@cardano-ogmios/client';
+
+export interface EraSummary {
+  parameters: {
+    epochLength: number;
+    slotLength: number;
+  };
+  start: {
+    slot: number;
+    time: Date;
+  };
+}
+
+export interface CardanoNode {
```

Comment:
I think it would be better to put all these new CardanoNode-related types under `core/src/CardanoNode`, because:
- `core/src/Cardano` will most likely end up having just the types that map to new transaction CDDL
- imho using it as `CardanoNode` is nicer than `Cardano.CardanoNode`

Reply:
Looking good, let's just move `packages/core/src/Cardano/types/EraSummary.ts` as well.